.vscode/launch.json

```diff
@@ -291,8 +291,8 @@
-              "NEW_DEPS_VERSION": "7.0.0-preview.23475.5",
+              "NEW_DEPS_URLS": "https://download.visualstudio.microsoft.com/download/pr/2ec8c3c2-4f6f-42c0-8c90-97fd97427d7c/59e252ec1565aa1e72aab4df73f35510/razorlanguageserver-linux-arm64-7.0.0-preview.23509.2.zip,https://download.visualstudio.microsoft.com/download/pr/2ec8c3c2-4f6f-42c0-8c90-97fd97427d7c/565066c5ca0844edb90b66187e99bd03/razorlanguageserver-linux-musl-arm64-7.0.0-preview.23509.2.zip,https://download.visualstudio.microsoft.com/download/pr/2ec8c3c2-4f6f-42c0-8c90-97fd97427d7c/c2c5dbc26f3ad91ac851ce175abb57e1/razorlanguageserver-linux-musl-x64-7.0.0-preview.23509.2.zip,https://download.visualstudio.microsoft.com/download/pr/2ec8c3c2-4f6f-42c0-8c90-97fd97427d7c/7c1e301d9000f06debecc8208d7aa2ea/razorlanguageserver-linux-x64-7.0.0-preview.23509.2.zip,https://download.visualstudio.microsoft.com/download/pr/2ec8c3c2-4f6f-42c0-8c90-97fd97427d7c/7b231eee6608879bc934a8256324e9f5/razorlanguageserver-osx-arm64-7.0.0-preview.23509.2.zip,https://download.visualstudio.microsoft.com/download/pr/2ec8c3c2-4f6f-42c0-8c90-97fd97427d7c/101e1db25b9411892eb6474c516a38b4/razorlanguageserver-osx-x64-7.0.0-preview.23509.2.zip,https://download.visualstudio.microsoft.com/download/pr/2ec8c3c2-4f6f-42c0-8c90-97fd97427d7c/59830fe8f7d57758675cc36f48b65a29/razorlanguageserver-win-arm64-7.0.0-preview.23509.2.zip,https://download.visualstudio.microsoft.com/download/pr/2ec8c3c2-4f6f-42c0-8c90-97fd97427d7c/22969b5b81fda9d5f5403b6cef265058/razorlanguageserver-win-x64-7.0.0-preview.23509.2.zip,https://download.visualstudio.microsoft.com/download/pr/2ec8c3c2-4f6f-42c0-8c90-97fd97427d7c/5fe01999736b69133fa1ea97fb35aed2/razorlanguageserver-win-x86-7.0.0-preview.23509.2.zip",
+              "NEW_DEPS_VERSION": "7.0.0-preview.23509.2",
```

Comment:
@allisonchou is it cool to just update the server? or do I need to update the telemetry too? Before this change, they were on the same version.

Reply:
I haven't tried updating just the server before so I'm not 100% sure, but I think updating solely the server should be fine as long as there are no telemetry-specific changes. I would verify there are no problems when opening a Razor file just to be sure though

Reply:
Opening razor files works fine, and i don't see the word "telemetry" in any worrying places in the Razor log, so I'm going to assume this all works 😁